### PR TITLE
Meals index

### LINKED ISSUE #10: Visitor can get all Meals in the database and their Foods
When a visitor makes a get request to /api/v1/meals, they get a list of all the Meals in the database and their associated Food items. The response's status is 200. 

Example response:
```
status: 200
body:
[
    {
        "id": 1,
        "name": "Breakfast",
        "foods": [
            {
                "id": 1,
                "name": "Banana",
                "calories": 150
            },
            {
                "id": 6,
                "name": "Yogurt",
                "calories": 550
            },
            {
                "id": 12,
                "name": "Apple",
                "calories": 220
            }
        ]
    },
    {
        "id": 2,
        "name": "Snack",
        "foods": [
            {
                "id": 1,
                "name": "Banana",
                "calories": 150
            },
            {
                "id": 9,
                "name": "Gum",
                "calories": 50
            },
            {
                "id": 10,
                "name": "Cheese",
                "calories": 400
            }
        ]
    },
    {
        "id": 3,
        "name": "Lunch",
        "foods": [
            {
                "id": 2,
                "name": "Bagel Bites - Four Cheese",
                "calories": 650
            },
            {
                "id": 3,
                "name": "Chicken Burrito",
                "calories": 800
            },
            {
                "id": 12,
                "name": "Apple",
                "calories": 220
            }
        ]
    },
    {
        "id": 4,
        "name": "Dinner",
        "foods": [
            {
                "id": 1,
                "name": "Banana",
                "calories": 150
            },
            {
                "id": 2,
                "name": "Bagel Bites - Four Cheese",
                "calories": 650
            },
            {
                "id": 3,
                "name": "Chicken Burrito",
                "calories": 800
            }
        ]
    }
]
```

### PR DESCRIPTION
**Pull request by teresa-m-knowles**
Closes #10 
Screen Shot 2019-05-09 at 10 09 02 AM

### CODE REVIEW

File: routes/api/v1/meals.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  include: [
+    {
+    model: Food,
+    as: 'foods',
+    attributes: ["id", "name", "calories"],
+    through: {
+      attributes: []
+    }
+    }]
```
Comment (timnallen) [inline routes/api/v1/meals.js:20]: So, adding the through and setting the attributes as empty removes the object. That's a cool workaround.

**Review by timnallen — APPROVED** [review]
Great work, Teresa, as always!

File: routes/api/v1/meals.js (hunk 1 above)
Comment (teresa-m-knowles) [reply routes/api/v1/meals.js:20]: Yes! Saw it in the comments sections here: https://gist.github.com/zcaceres/83b554ee08726a734088d90d455bc566

**Review by timnallen — APPROVED** [review]
Sweet!

**Pull request merged**